File: class_tool/src/main/java/com/tempsfteam/class_tool/entity/PermissionInfo.java
package com.tempsfteam.class_tool.entity;

import lombok.AllArgsConstructor;
import lombok.Data;
import lombok.NoArgsConstructor;

/**
 * @author ADACHI
 */
@Data
@NoArgsConstructor
@AllArgsConstructor
public class PermissionInfo {
    // 角色id
    private Integer roleId;
    // 权限id
    private Integer permissionId;
    // 权限对应模块
    private String module;
    // 权限对应英文介绍
    private String description;
    // 权限对应中文介绍
    private String desCN;
}

File: class_tool/src/main/resources/static/src/models/entity/PermissionInfo.js
export default {
    name: 'permissionInfo',
    title: 'PermissionInfo',
    fields: [
        { name: 'roleId', label: '角色id', type: 'number' },
        // 嵌套 permission 对象（按仓库风格仅在 schema 中声明字段名称）
        { name: 'permissionId', label: '权限id', type: 'number' },
        { name: 'module', label: '权限对应模块', type: 'string' },
        { name: 'description', label: '权限对应英文介绍', type: 'string' },
        { name: 'desCN', label: '权限对应中文介绍', type: 'string' }
    ]
};